docparse/chunking_tutorial.mdx

```diff
@@ -0,0 +1,87 @@
+
+```
+chunking_options={
+  "strategy": "context_rich",
+  "tokenizer": "openai_tokenizer",
+  "tokenizer_options": {
+    "model_name": "text-embedding-3-small"
+  },
+  "merge_across_pages": True,
+  "max_tokens": 512,
+}
+
+with open("transformers.pdf", "rb") as f:
+  data = partition_file(f, aryn_api_key, chunking_options=chunking_options)
+  
+
+```
+
+If you inspect the return value, you’ll notice that the section header and the formula are all chunked together:
+
+```python
+{
+"properties": {
+"score": 0.9430291056632996,
+"page_number": 5,
+"page_numbers": [
+5
+]
+},
+"type": "Text",
+"binary_representation": null,
+"text_representation": 
+"**3.3 Position-wise Feed-Forward Networks**\n\nIn addition to attention sub-layers, each of the layers in our encoder and decoder contains a fully\nconnected feed-forward network, which is applied to each position separately and identically. This\nconsists of two linear transformations with a ReLU activation in between.\n\n
+**FFN(x) = max(0, xW1 + b1)W2 + b2**\n (2)\n\nWhile the linear transformations are the same across different positions, they use different parameters\nfrom layer to layer. Another way of describing this is as two convolutions with kernel size 1.\nThe dimensionality of input and output is dmodel = 512, and the inner-layer has dimensionality\ndf f = 2048.\n",
+"bbox": [
+0.17546135397518384,
+0.6537761896306818,
+0.8272261316636029,
+0.8000078790838068
+],
+"_header": "3.3 Position-wise Feed-Forward Networks\n"
+}
+```
+
+## Maximize Within Limit
+
+The Maximize within limit strategy is meant to be used when you want to merge several consecutive elements together into a large chunk. Take the following example: 
+
+<Frame>
+  <img src="../images/intro_algorithms.png" />
+</Frame>
+
+
+the bounding boxes shown above are the result of calling DocParse without any chunking options specified. Let’s say for your question/answering RAG application, you want all the list items to be grouped together into 1 chunk to improve the quality of your embedding. Calling DocParse with the following chunking options will group the entire list into 1 chunk: 
+
+```
+chunking_options={
+  "strategy": "maximize_within_limit",
+  "tokenizer": "openai_tokenizer",
+  "tokenizer_options": {
+    "model_name": "text-embedding-3-small"
+  },
+  "merge_across_pages": True,
+  "max_tokens": 512,
+}
+
+with open("transformers.pdf", "rb") as f:
+  data = partition_file(f, aryn_api_key, chunking_options=chunking_options)
```

Comment:
Where is the output for this?

Reply:
The output was too large so I didn't include them here.